content/en/docs/project-user-guide/alerting/alerting-message.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Alerting Message - workload level"
+keywords: 'KubeSphere, Kubernetes, Workload, Alerting, Message, Notification'
+description: 'Alerting Message - workload level'
+
+linkTitle: "Alerting Message - workload level"
+weight: 2220
+---
+
+Alert Message records the detailed information of alert triggered by alert rule. In it, you can view alert details, monitoring targets, alert policies, recent notifications, comments and etc.
+
+## Prerequisites
+
+You have created a workload-level alert policy and receive alert notifications of it. If no, please refer to [Alert Policy - workload level](../alerting-policy/) to create one.
+
+## Hands-on Lab
+
+### Task 1: View Alert Message
+
+1. Log in the console and go to your project. Navigate to **Alerting Message** under **Monitoring & Alerting**, and you can see alert messages in the list. In the example of [Alert Policy - workload level](../alerting-policy/) you set two monitoring targets `reviews-v1` and `details-v1`, and both of their memory usages are higher than the threshold, so you can see two alert messages corresponding to them.
+
+![alerting_message_workload_level_list](/images/docs/alerting/alerting_message_workload_level_list.png)
+
+2. Select one of the alert messages to enter the details page, and in **Alert Detail** you can see the graph of the memory usage of the monitored workload over time, which has been continuously higher than the threshold of 20 MiB set by the alert rule, so the alert was triggered.
+
+![alerting_message_workload_level_detail](/images/docs/alerting/alerting_message_workload_level_detail.png)
+
+### Task 2: View Alert Policy
+
+Switch to **Alert Policy** to view the alert policy corresponding to this alert message, and you can see triggering rule of it set in the example of [Alert Policy - workload level](../alerting-policy/).
```

Comment:
->
Switch to **Alerting Policy** to view the alert policy corresponding to this alert message, and you can see the triggering rule of it set in the example of [Alert Policy - workload level](../alerting-policy/).